js/phemex.js

```diff
@@ -2020,6 +2026,13 @@ module.exports = class phemex extends Exchange {
         return this.parseOrder (data, market);
     }
 
+    async createReduceOnlyOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        const request = {
+            'reduceOnly': true,
+        };
+        return await this.createOrder (symbol, type, side, amount, price, this.extend (request, params));
+    }
+
```

Comment:
We should probably add this to `base.exchange` and implement it the same as `createPostOnlyOrder` so that we stop writing the same method on every exchange

Reply:
I'll work on this and open a separate PR